cabal.project

```diff
@@ -51,3 +51,41 @@ packages:
 tests: True
 optimization: False
 -- reorder-goals: True
+
+allow-newer: all
+
+-- https://github.com/GaloisInc/cereal/pull/103
+source-repository-package
+    type: git
+    location: https://github.com/parsonsmatt/cereal
+    tag: b4acf0b778a1d5da305f4de013ce7f32b53d8282
```

Comment:
this can be removed, see https://github.com/GaloisInc/cereal/commits/master

Reply:
Nice thanks